packages/wallets/src/features/cashier/modules/WithdrawalVerification/WithdrawalVerificationSent/__tests__/WithdrawalVerificationSent.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import WithdrawalVerificationSent from '../WithdrawalVerificationSent';
+
+describe('WithdrawalVerificationSent', () => {
+    jest.useFakeTimers();
```

Comment:
are timers needed in this component? the component itself seems to be synchronous, the parent component provides alle the async stuff / timed stuff like counters etc,

Reply:
Ah you're right, I was trying to imitate the behaviour of the counter but it works without the timer, updated 👍